renovate/default.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "name": "renovate-mui-config",
+  "description": "Renovate configuration for MUI repositories.",
+  "automerge": false,
+  "commitMessageAction": "Bump",
+  "commitMessageExtra": "to {{newValue}}",
+  "commitMessageTopic": "{{depName}}",
+  "dependencyDashboard": true,
+  "rebaseWhen": "conflicted",
+  "labels": ["dependencies"],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 6:00am on the first day of the month"
+  },
+  "postUpdateOptions": ["pnpmDedupe"],
+  "prConcurrentLimit": 30,
+  "prHourlyLimit": 0,
+  "rangeStrategy": "bump",
+  "schedule": "on sunday before 6:00am",
+  "timezone": "UTC",
+  "ignorePaths": ["examples/**"],
```

Comment:
Why are examples excluded? Don't we want to keep their dependendies up to date?

Reply:
All our examples generally have `latest` as the dependency version.

Reply:
So far, only `material-ui` has this. Other repos don't contain `examples` as such. Also, `material-ui`'s examples aren't part of pnpm workspace. So they wouldn't be affected by renovate anyway.
So, maybe its best to move `ignorePaths` to that repo itself. Based on future development, we can check if it needs to be here.